src/Components/Nav/Nav.js

```diff
@@ -1,9 +1,79 @@
+/*eslint-disable*/
 import React, { Component } from 'react';
+import InnerNav from './InnerNav/InnerNav';
+import NAV_DATA from './navData';
 import './Nav.scss';
 
 class Nav extends Component {
   render() {
-    return <div></div>;
+    return (
+      <header className="header">
+        <nav className="nav">
+          <div className="navWrap">
+            <div className="logoWrap">
+              <a className="MLBlogo">
+                <h1>MLB</h1>
+              </a>
+            </div>
+            <div className="menuLists">
+              <ul className="list">
+                {NAV_DATA.map(item => {
+                  const { id, name, link, cate, depth, banner } = item;
```

Comment:
```suggestion
                {NAV_DATA.map(({ id, name, link, cate, depth, banner }) => {
```
매개변수자리에서 바로 구조분해할당 가능합니다

Reply:
띠용... 적용하겠습니다.